Cargo.toml

```diff
@@ -174,6 +174,7 @@ symbolic-unreal = { version = "12.1.2", default-features = false }
 syn = "1.0.14"
 syn2 = { package = "syn", version = "2.0.11" }
 synstructure = "0.12.3"
+synstructure2 = { package = "synstructure", version = "0.13.1" }
```

Comment:
Why are two versions of this package required?

Reply:
We still have `relay-event-derive` which depends on `syn 1` and therefore `synstructure 0.12`. After the other one is also updated, we can clean this up.